Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkBehavior.cs

```diff
@@ -22,10 +22,10 @@ public abstract class NetworkBehavior : MonoBehaviour, INetworkBehavior
 
 		protected virtual void NetworkStart()
 		{
+			CompleteRegistration();
```

Comment:
I think that this is a good idea but may cause another issue because of the code found here:  https://github.com/mcguinnessdr/ForgeNetworkingRemastered/blob/f78c9ec3977db3b3b40bbdbad321c471911bbf69/BeardedManStudios/Source/Forge/Networking/NetWorker.cs#L497

The actions should be executed after the network start. Those actions are the actions that are pending for the objects creation (such as RPC calls or buffered calls that were done before the object was finally finished being created). I may make another branch to fix this and merge your branch into it